packages/webapp/src/components/Crop/ManagementDetail/styles.module.scss

```diff
@@ -60,3 +61,51 @@
 .rating {
   display: table-row;
 }
+
+.menuIcon {
+  width: 48px;
+  font-size: 20px;
+  color: var(--grey600);
+  cursor: pointer;
+}
+
+.copyRepeatMenu {
+  position: absolute;
+  right: 0;
+  top: 100%;
+  display: flex;
+  width: 260px;
+
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  flex-shrink: 0;
+  gap: -4px;
```

Comment:
![Screenshot 2023-07-26 at 11 06 33 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/7dc5dad3-6e4f-4ba0-9eda-751ea563f76b)
I'm not very familiar with flex, but `gap` should be a positive value?

Reply:
That... shouldn't be there!